### DISCUSSION TITLE
Oqtane Development & Multi-Tenant

### DISCUSSION THREAD

**Discussion by pnicosia80** [oqtane/oqtane.framework#3259] [Q&A]
Good morning,
I'm developing modules for Oqtane in Visual Studio 2022 and to run Oqtane I launch IIS Express from Visual Studio. Now I created a new tenant and I see it on db, but I can't navigate to the new tenant to which I assigned a different port on localhost (Error 404). How can I solve it??

If I wanted to install everything on IIS using the Oqtane source folder, how can I do it?

Thank you

Greetings

**Comment by vnetonline** [comment]
@pnicosia80 i suggest you use [Oqtane.me](https://oqtane.me) have a look there you should be able to find the resources you need to setup Oqtane and use *.oqtane.me to access the tenants you setup as long as you give them *.Oqtane.me alias.

**Comment by sbwalker** [comment]
@pnicosia80 in addition to what @vnetonline mentioned above, if you simply want to test the multi-tenant capabilities in a local development environment you can use sub-folders to identify a site ie. localhost:44358/site1 and localhost:44358/site2

**Reply by Bhushan02-dot** [reply]
@sbwalker  in local development you suggest sub-folder for multitenant  but we want different ports then how ?
